tests/config/test_find_default_config_files.py

```diff
@@ -129,6 +129,49 @@ def test_pylintrc_parentdir() -> None:
             assert next(config.find_default_config_files()) == expected
 
 
+@pytest.mark.usefixtures("pop_pylintrc")
+def test_pyproject_toml_parentdir() -> None:
+    """Test the search of pyproject.toml file in parent directories"""
+    with tempdir() as chroot:
+        with fake_home():
+            chroot_path = Path(chroot)
+            files = [
+                "pyproject.toml",
+                "git/pyproject.toml",
+                "git/a/pyproject.toml",
+                "git/a/.git",
+                "git/a/b/c/__init__.py",
+                "hg/pyproject.toml",
+                "hg/a/pyproject.toml",
+                "hg/a/.hg",
+                "hg/a/b/c/__init__.py",
+                "none/sub/__init__.py",
+            ]
```

Comment:
Shouldn't 'files' and 'expected' be in a ``@pytest.mark.parametrize`` instead ?

Reply:
I don't know, sorry. It's the same was as `test_pylintrc_parentdir` is tested RN. Can we consider refactoring both tests later, maybe?

Reply:
I did not realize that it's the exact same list than another test. We might want to refactor before the change and use the same constant in both, then ?

Reply:
It's not used anywhere else. It's just the same way of testing that is used in other places in this file. I don't see any other example of `files` and `expected` being a `@pytest.mark.parametrize`

See `test_pylintrc_parentdir`, `test_pylintrc_parentdir_no_package`, `test_verbose_output_no_config`, and `test_verbose_abbreviation`

Reply:
I'm going to resolve then.